src/applications/financial-status-report/components/ItemLoop.jsx

```diff
@@ -199,8 +199,13 @@ const AddAnotherButton = ({ uiOptions, handleAdd, collapsed }) => {
     <div>
       <div className="add-item-container" name="table_root_">
         <div className="add-item-button-section">
-          <button className={linkClassNames} onClick={handleAdd} type="button">
-            <i className="fas fa-plus plus-icon" />
+          <button
+            className={linkClassNames}
+            onClick={handleAdd}
+            type="button"
+            tabIndex={0}
```

Comment:
I'm not sure why `tabIndex` is necessary here. Shouldn't it already be tab-able since it is a button?

Reply:
@mlauren Nice catch, you're right. I removed the tabindex property.